### PR TITLE
Fixes. Refactor.

### PR DESCRIPTION
**Pull request by gtsonevv**

### CODE REVIEW

File: apps/backend/src/allowlist/allowlist.service.ts [hunk 1]
```diff
@@ -102,71 +115,64 @@ export class AllowlistService {
...
+    );
+  }
+
+  private async passCheck(url, target) {
+    let arr = [];
 
     let res;
+    let next_token;
     while (true) {
+      const params = { max_results: 100 };
```
Comment (maptuhec) [inline apps/backend/src/allowlist/allowlist.service.ts:153]: this check can go to infinity with the `true` hardcoded, can you think of a better way?

File: apps/frontend/src/core/utilities/ProjectUtils.tsx [hunk 2]
```diff
@@ -1,153 +1,198 @@
...
+    inputN.style.overflow = 'hidden';
+    inputN.style.opacity = '0';
+    inputN.value = text;
+    document.body.appendChild(inputN);
+    ProjectUtils.copyInput(inputN);
+    document.body.removeChild(inputN);
+  }
+
+  static downloadUrl(url: string, filename = S.Strings.EMPTY) {
+    const a = document.createElement('a');
```
Comment (maptuhec) [inline apps/frontend/src/core/utilities/ProjectUtils.tsx]: what is `a`, can there be a more meaningful naming

File: apps/frontend/src/features/allowlists/presentation/components/allowlist.tsx [hunk 3]
```diff
@@ -1,12 +1,31 @@
...
+
+    const data = await ProjectUtils.signMessage(
+      CHAIN_DETAILS.RPC_ADDRESS[props.walletStore.selectedNetwork],
+      props.walletStore.ledger.offlineSigner,
+      props.walletStore.getAddress(),
+    );
+
+    try {
+      const res = await axios.post(url, data);
+      alert('success');
```
Comment (maptuhec) [inline apps/frontend/src/features/allowlists/presentation/components/allowlist.tsx:23]: is this for testing purposes only?

File: apps/frontend/src/features/allowlists/presentation/components/create-allowlist-form.tsx [hunk 4]
```diff
@@ -52,8 +54,19 @@ const CreateAllowlistForm = () => {
...
+    const signData = await ProjectUtils.signMessage(
+      CHAIN_DETAILS.RPC_ADDRESS[walletStore.selectedNetwork],
+      walletStore.ledger.offlineSigner,
+      walletStore.getAddress(),
+    );
+
+    data = { ...data, ...signData };
+    try {
+      await axios.post(url, data);
+      alert('success');
```
Comment (maptuhec) [inline apps/frontend/src/features/allowlists/presentation/components/create-allowlist-form.tsx:66]: same here?

File: apps/frontend/src/features/allowlists/presentation/pages/my-allowlists.tsx [hunk 5]
```diff
@@ -1,14 +1,19 @@
...
-import Allowlist from '../components/allowlist-preview';
+import Allowlist from '../components/allowlist';
 
-function MyAllowlistsPage() {
+function MyAllowlistsPage({ walletStore }) {
   const [allowlists, setAllowlists] = useState([]);
 
   useEffect(() => {
     (async () => {
+      const address = window.localStorage.getItem('addr').split('"').join('');
```
Comment (maptuhec) [inline apps/frontend/src/features/allowlists/presentation/pages/my-allowlists.tsx]: which address is this?

**Review by maptuhec — CHANGES_REQUESTED** [review]

File: apps/frontend/src/features/allowlists/presentation/components/allowlist.tsx (hunk 3 above)
Comment (gtsonevv) [reply apps/frontend/src/features/allowlists/presentation/components/allowlist.tsx:23]: yes, this alert is for testing purposes only

**Review by maptuhec — APPROVED** [review]

**Pull request merged**